### PR TITLE
Added a cataloger for PHP PECL Packages

### PR DESCRIPTION
**Pull request by LaurentGoderre**
Add support for PHP PECL and PEAR extensions: https://pecl.php.net/

### CODE REVIEW

File: syft/pkg/php.go [hunk 1]
```diff
@@ -36,3 +36,9 @@ type PhpComposerAuthors struct {
 	Email    string `json:"email,omitempty"`
 	Homepage string `json:"homepage,omitempty"`
 }
+
+type PhpPeclEntry struct {
+	Name    string
+	Version string
+	License []string
```
Comment (kzantow) [inline syft/pkg/php.go:43]: These fields should have `json` tags with lower camelCase ,e.g.:
```
Name string `json:"name"`
Version string `json:"version"`
License string `json:"license"`
```

**Review by kzantow — COMMENTED** [review]
Thanks @LaurentGoderre! I noted a couple of small things to update and the rest looks good; shout if you need help with any of it

**Comment by LaurentGoderre** [comment]
@kzantow Updated

**Review by kzantow — COMMENTED** [review]
Thanks @LaurentGoderre! The schema looks good now. I left a few more comments about the type assertions -- these will need to get updated. Consider the other comments non-blocking

**Comment by LaurentGoderre** [comment]
@kzantow updated

**Review by kzantow — APPROVED** [review]
LGTM, thanks much @LaurentGoderre !

**Pull request merged**